src/policy/fees.cpp

```diff
@@ -87,73 +96,143 @@ double TxConfirmStats::EstimateMedianVal(int confTarget, double sufficientTxVal,
     int maxbucketindex = buckets.size() - 1;
 
     // requireGreater means we are looking for the lowest fee/priority such that all higher
-    // values pass, so we start at maxbucketindex (highest fee) and look at successively
+    // values pass, so we start at maxbucketindex (highest fee) and look at succesively
     // smaller buckets until we reach failure.  Otherwise, we are looking for the highest
```

Comment:
the original was spelled correctly.

Reply:
i dont think i actually meant to do that. was probably a key smash in vim. will fix